entities.js

```diff
@@ -47,17 +47,14 @@ $(document).ready(function(){
     });
 });
 
-// Smooth Anchor scrolling
 $(document).ready(function() {
-
-    $('a[href*=#]').each(function() {
-        if($(this).attr('href').indexOf("#") == 0) {
-            $(this).click(function(e) {
-              e.preventDefault();
-              var targetOffset = $($(this).attr('href')).offset().top;
-              $('body').animate({scrollTop: targetOffset - 75}, 500);
-            });
-        }
-    });
+	
+	// Smooth Anchor scrolling
+	$('a[href^=#]').click(function(){
+		$('html, body').animate({
+			scrollTop: $( $(this).attr('href') ).offset().top
```

Comment:
@redpik 

Can you manually add some height to the offset. I remember adding it to displace the title's height so we can see that starting section.

And thanks for the addition of the white triangles.